client-src/elements/chromedash-guide-editall-page.js

```diff
@@ -102,7 +104,9 @@ export class ChromedashGuideEditallPage extends LitElement {
       if (hash) {
         const el = this.shadowRoot.querySelector(hash);
         if (el) {
-          this.shadowRoot.querySelector(`chromedash-form-field[name="${el.name}"] tr th b`).scrollIntoView(true, {behavior: 'smooth'});
+          const formField = this.shadowRoot
```

Comment:
Dan is rewriting this functionality in 
https://github.com/GoogleChrome/chromium-dashboard/pull/2804

I'll ask him to look at what you are doing here and incorporate the same case logic.

Reply:
It looks like the change here is mostly just reformatting, except for the (formField || el).  Is the "formField", which is really only the header associated with the form field, sometimes missing? 

I suppose we have a conflict between my change and this change, both unsubmitted at the moment.  Is it OK to leave this line unchanged for now, or does it cause an error if the "formField" element (really the header) is missing?